hera_cal/reflections.py

```diff
@@ -1287,14 +1287,14 @@ def L(x, amp, dly, phs, clean_data, clean_model, clean_wgts, dly_range, freqs, f
     if clean_flags is None:
         clean_wgts = np.ones_like(clean_data)
     else:
-        clean_wgts = (~clean_flags).astype(np.float)
+        clean_wgts = (~clean_flags).astype(float)
     edge_flags = np.isclose(np.mean(clean_wgts, axis=0), 0.0)
     edge_flags[np.argmin(edge_flags):-np.argmin(edge_flags[::-1])] = False
     if fix_amp and fix_dly and fix_phs:
         raise ValueError("Can't hold amp, dly and phs fixed")
 
     # create delay range
-    if isinstance(dly_range, (int, np.int, float, np.float)):
+    if isinstance(dly_range, (int, int, float, float)):
```

Comment:
I'm not sure if this is what you want to do? Above, I see something like `if isinstance(x, (int, np.integer, float, np.floating))`, which is what I imagine you're going for. One thing I'm wondering is whether you can get away with just doing `if isinstance(x, (int, float))`.

Reply:
Good catch. We want np.floating and np.integer to catch other float/int types like float32 or whatever. Fixed.